Code/Framework/AzCore/Tests/Asset/AssetManagerLoadingTests.cpp

```diff
@@ -365,6 +365,40 @@ namespace UnitTest
 
     };
 
+    template <typename Pred>
+    bool DispatchEventsUntilCondition(AZ::Data::AssetManager& assetManager, Pred&& conditionPredicate,
+        AZStd::chrono::seconds logIntervalSeconds = BaseAssetManagerTest::DefaultTimeoutSeconds,
+        AZStd::chrono::seconds maxTimeoutSeconds = BaseAssetManagerTest::DefaultTimeoutSeconds * 12)
```

Comment:
magic number `12`. Same magic number used in other places. Please assign it to a const variable with a descriptive name.